ui/page/debug_page.go

```diff
@@ -13,47 +13,51 @@ import (
 const DebugPageID = "Debug"
 
 type debugItem struct {
-	text   string
-	page   string
-	action func()
+	text      string
+	page      string
+	action    func()
+	clickable *decredmaterial.Clickable
 }
 
 type DebugPage struct {
 	*load.Load
+	container  layout.Flex
+	shadowBox  *decredmaterial.Shadow
 	debugItems []debugItem
-	list       *decredmaterial.ClickableList
```

Comment:
Undo this. The clickable list has `IsShadowEnabled` Param. Use it instead see example use case here.
.https://github.com/planetdecred/godcr/blob/928443619efe397e68133e2692ca5ee1675bb315/ui/page/transaction/transactions_page.go#L51.

<img width="801" alt="image" src="https://user-images.githubusercontent.com/27733432/160543594-c5e84537-8ae9-4a56-ad0e-6ed380892223.png">